<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>690</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Учет заявок на ремонт оргтехники</string>
  </property>
  <property name="windowIcon">
   <iconset theme="accessories-dictionary">
    <selectedon>../../../../../logo.png</selectedon>
   </iconset>
  </property>
  <property name="toolTipDuration">
   <number>0</number>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
background-color: rgb(4, 160, 255);
color: rgb(255, 255, 255);
border: 0px;
font: 14pt
}
QWidget
{font:  &quot;Comic Sans MS&quot;;
}
</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#widget{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(4, 160, 255, 255), stop:1 rgba(231, 250, 191, 255));}

</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>150</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="pixmap">
         <pixmap>../media/logo.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Учет заявок на ремонт оргтехники</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="AvtorButton">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Выход</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="Avtorisation">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>50</number>
         </property>
         <property name="topMargin">
          <number>50</number>
         </property>
         <property name="rightMargin">
          <number>50</number>
         </property>
         <property name="bottomMargin">
          <number>50</number>
         </property>
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>12</number>
           </property>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <pointsize>24</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>АВТОРИЗАЦИЯ</string>
             </property>
             <property name="scaledContents">
              <bool>false</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Логин</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="LoginField"/>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>Пароль</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="PasswordField"/>
           </item>
           <item>
            <widget class="QPushButton" name="SignInButton">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Войти</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="ErrorField">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 0, 0);</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="user">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QTableWidget" name="tableMasteraZayavki">
             <property name="minimumSize">
              <size>
               <width>700</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_3">
                 <item>
                  <widget class="QLineEdit" name="lineEdit"/>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit_2"/>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit_3"/>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit_4"/>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit_5"/>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit_6"/>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit_7"/>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit_8"/>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit_9"/>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit_10"/>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QPushButton" name="edit_button">
                 <property name="text">
                  <string>редактировать</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="delete_button">
                 <property name="text">
                  <string>удалить</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="insert_button">
                 <property name="text">
                  <string>добавить</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
